src/test/HelmetTest.js

```diff
@@ -219,6 +232,7 @@ describe("Helmet", () => {
 
                 expect(htmlTag.getAttribute("lang")).to.be.null;
                 expect(htmlTag.getAttribute("amp")).to.be.null;
+                expect(htmlTag.getAttribute(HELMET_ATTRIBUTE)).to.equal(null);
```

Comment:
Is this correct? Shouldn't `htmlTag.getAttribute(HELMET_ATTRIBUTE)` equal `lang, amp` at this point?